test/test_ui_commands.py

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf8 -*-
+# This file is part of beets.
+# Copyright 2015, Adrian Sampson.
+#
+# Permission is hereby granted, free of charge, to any person obtaining
+# a copy of this software and associated documentation files (the
+# "Software"), to deal in the Software without restriction, including
+# without limitation the rights to use, copy, modify, merge, publish,
+# distribute, sublicense, and/or sell copies of the Software, and to
+# permit persons to whom the Software is furnished to do so, subject to
+# the following conditions:
+#
+# The above copyright notice and this permission notice shall be
+# included in all copies or substantial portions of the Software.
+
+"""Test module for file ui/commands.py
+"""
+import os
+import shutil
+
+from test import _common
+from test._common import unittest
+
+from beets import library
+from beets import ui
+from beets.ui import commands
+
+
+class QueryTest(_common.TestCase):
+    def setUp(self):
+        super(QueryTest, self).setUp()
+
+        self.io.install()
+
+        self.libdir = os.path.join(self.temp_dir, 'testlibdir')
+        os.mkdir(self.libdir)
+
+        # Add a file to the library but don't copy it in yet.
+        self.lib = library.Library(':memory:', self.libdir)
+
+        # Alternate destination directory.
+        self.otherdir = os.path.join(self.temp_dir, 'testotherdir')
+
+    def add_Item(self, filename='srcfile', templatefile='full.mp3'):
```

Comment:
In PEP8 style, methods are always lower case, even if they refer to something upper-casey. (Yes, setUp breaks this rule, sadly. :cry:)